scalatest-fs2/src/main/scala/com/dwolla/testutils/scalatest/ScalaTestCatsInstances.scala

```diff
@@ -0,0 +1,20 @@
+package com.dwolla.testutils.scalatest
+
+import cats._
+import cats.implicits._
+import fs2._
+import org.scalatest.enablers.Containing
+
+trait ScalaTestCatsInstances {
+  implicit def ContainingContravariant: Contravariant[Containing] = new Contravariant[Containing] {
+    override def contramap[A, B](fa: Containing[A])(f: B ⇒ A): Containing[B] = new Containing[B] {
+      override def contains(container: B, element: Any): Boolean = fa.contains(f(container), element)
+
+      override def containsOneOf(container: B, elements: Seq[Any]): Boolean = fa.containsOneOf(f(container), elements)
```

Comment:
I think it would be nice to have a test for `containsOneOf` and `containsNoneOf`.  The tests might be useful for future maintainers or for users of this library.

Reply:
Good idea, added in 689e9f6